cmd/object-handlers.go

```diff
@@ -2461,21 +2461,14 @@ func (api objectAPIHandlers) CompleteMultipartUploadHandler(w http.ResponseWrite
 
 	partsMap := make(map[string]PartInfo)
 	if isEncrypted {
-		var partNumberMarker int
-		maxParts := 1000
-		for {
-			listPartsInfo, err := objectAPI.ListObjectParts(ctx, bucket, object, uploadID, partNumberMarker, maxParts, opts)
-			if err != nil {
-				writeErrorResponse(ctx, w, toAPIError(ctx, err), r.URL, guessIsBrowserReq(r))
-				return
-			}
-			for _, part := range listPartsInfo.Parts {
-				partsMap[strconv.Itoa(part.PartNumber)] = part
-			}
-			partNumberMarker = listPartsInfo.NextPartNumberMarker
-			if !listPartsInfo.IsTruncated {
```

Comment:
Are we removing this because, `IsTruncated` will never be `True` as the maxParts was set to maximum?

Reply:
There cannot be more than 10k parts. I imagine that is why it doesn't need to be checked.